ci-operator/step-registry/ipi/conf/nutanix/gpu/ipi-conf-nutanix-gpu-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-gpu.yaml"
+
+NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
+declare gpu_name
+declare gpu_device_id
```

Comment:
I wonder where created the var ?

Reply:
They are in vault openshift-qe/nutanix-qe-gpu/secrets